examples/connectivity/plot_ica_neurovault.py

```diff
@@ -0,0 +1,244 @@
+    338, 339,                      # And the following are crap
+    335,                           # 335 is a duplicate of 336
+    3360, 3362, 3364,              # These are mean images, and not Z maps
+    1202, 1163, 1931, 1101, 1099]  # Ugly / obviously not Z maps
+imfilts = [lambda im: im.get('perc_bad_voxels', 0.) < 100.]
+
+# Download 100 matching images
+ss_all = datasets.fetch_neurovault(max_images=100,  # Use np.inf for all imgs.
+                                   collection_ids=[-bid for bid in bad_collects],
+                                   image_ids=[-bid for bid in bad_image_ids],
+                                   map_types=['F map', 'T map', 'Z map'],
+                                   image_filters=imfilts)
+images, collections = ss_all['images'], ss_all['collections']
+
+# -------------------------------------------
+# Resample the images
+target_img = datasets.load_mni152_template()
+resampled_niis = []
+for ii, image in enumerate(images):
+    if ii % 100 == 0:
+        print("Resampling image %d-%d of %d..." % (
+            ii + 1, min(ii + 100, len(images)), len(images)))
+    src_img = nib.load(image['local_path'])
+    file_path, ext = splitext_multi(image['local_path'])
+    resample_path = '%s-resampled%s' % (file_path, ext)
+    resampled_niis += resample_and_expand_image(src_img, target_img,
+                                                resample_path)
+if len(images) != len(resampled_niis):
+    print("After resampling, %d images => %d "
+          "(each time point became an image)" % (
+              len(images), len(resampled_niis)))
+
+# -------------------------------------------
+# Get the neurosynth terms; returns a dict of terms, with
+#   a vector of values (1 per image)
+terms = get_neurosynth_terms(images, _get_dataset_dir('neurosynth'))
+good_terms = dict([(t, v) for t, v in terms.items()
+                   if np.sum(v[v > 0]) > 0.])
+
+print("Top 100 neurosynth terms:")
+sort_idx = np.argsort([np.sum(v[v > 0]) for v in good_terms.values()])
+for term_idx in sort_idx[-100:]:
+    # Eliminate negative values
+    term = list(good_terms.keys())[term_idx]
+    vec = good_terms[term]
+    vec[vec < 0] = 0
+    print('\t%-25s: %.4e' % (term, np.sum(vec)))
+
+# -------------------------------------------
+# Get the grey matter mask. Cheat by pulling from github :)
+print("Downloading and resampling grey matter mask.")
+url = 'https://github.com/NeuroVault/neurovault_analysis/raw/master/gm_mask.nii.gz'
+mask = _fetch_files(_get_dataset_dir('neurovault'),
+                    (('gm_mask.nii.gz', url, {}),))[0]
+mask = resample_and_expand_image(nib.load(mask), target_img)[0]
+mask = nib.Nifti1Image((mask.get_data() >= 0.5).astype(int),
+                       mask.get_affine(), mask.get_header())
+
+# -------------------------------------------
+# Do the ICA transform.
+masker = NiftiMasker(mask_img=mask, memory=mem)
+X = masker.fit_transform(resampled_niis)
```

Comment:
I am wondering: maybe we can do the resampling using the NiftiMasker's resampling functionality? If that's possible, it would simplify the example.